Sources/Build/BuildDescription/SwiftTargetBuildDescription.swift

```diff
@@ -644,6 +626,67 @@ public final class SwiftTargetBuildDescription {
 
         return args
     }
+    
+    /// Determines the arguments needed to run `swift-symbolgraph-extract` for
+    /// this module.
+    public func symbolGraphExtractArguments() throws -> [String] {
+        var args = [String]()
+        args += try self.cxxInteroperabilityModeArguments(
+            propagateFromCurrentModuleOtherSwiftFlags: true)
+        return args
```

Comment:
Can this be simplified to just `return self.cxxInteroperabilityModeArguments(propagateFromCurrentModuleOtherSwiftFlags: true)`?